### PR TITLE
scaliform formated source code

### PR DESCRIPTION
**Pull request by MansurAshraf**
Pure scalariform formatted source without any code change.

### CODE REVIEW

**Comment by ianoc** [comment]
Is this right ? A simple compile doesn't produce this diff which runs
scalariform ?

**Comment by johnynek** [comment]
I thought we turned off the rule that leaves the trailing `)` on newline.

This looks strange.

File: scalding-args/src/main/scala/com/twitter/scalding/Args.scala [hunk 1]
```diff
@@ -41,16 +41,17 @@ object Args {
...
-        .foldLeft(List("" -> List[String]())) { (acc, arg) =>
-          val noDashes = arg.dropWhile{ _ == '-' }
-          if (arg == noDashes || isNumber(arg))
-            (acc.head._1 -> (arg :: acc.head._2)) :: acc.tail
-          else
-            (noDashes -> List()) :: acc
-        }
-        //Now reverse the values to keep the same order
-        .map { case (key, value) => key -> value.reverse }.toMap)
+      .filter{ a => !a.matches("\\s*") }
```
Comment (jnievelt) [inline scalding-args/src/main/scala/com/twitter/scalding/Args.scala]: Is this catching some "multiple lines in a function application should be aligned" thing?  If so, maybe declare val processedArgs = args... ahead of this, so the processing chain isn't inside the Args()

**Comment by MansurAshraf** [comment]
This was against a branch cut from develop with no code change. Can someone else verify that its happening for them as well? I talked to @Gabriel439 and he had the same issue

**Comment by ianoc** [comment]
I ran a compile on a local develop and it had no changes.

**Comment by Gabriel439** [comment]
I got scalariform-related changes when I did `twitter_release` to prepare our internal release

**Comment by Gabriel439** [comment]
So it turns out that the kind of changes that Mansur was talking about were much larger than the ones I saw (just a few small changes).  He identified it as a problem specific to his dev environment

**Comment by MansurAshraf** [comment]
It was https://github.com/ensime/ensime-sbt plugin in my global setting causing the problem

**Pull request closed without merging**